plugin/src/sbt-test/sbt-mu-srcgen/basic/build.sbt

```diff
@@ -1,5 +1,10 @@
 version := sys.props("version")
 
-libraryDependencies ++= Seq(
-  "io.higherkindness" %% "mu-rpc-server" % sys.props("mu")
-)
+lazy val root = project
+  .in(file("."))
+  .enablePlugins(SrcGenPlugin)
+  .settings(Seq(
+    libraryDependencies ++= Seq(
+  "io.higherkindness" %% "mu-rpc-server" % sys.props("mu"),
+  )
+  ))
```

Comment:
Actually, I think you can enable the plugin as follows:

```scala
version := sys.props("version")

enablePlugins(SrcGenPlugin)

libraryDependencies ++= Seq(
  "io.higherkindness" %% "mu-rpc-server" % sys.props("mu")
)
```

Reply:
oh nice that's way easier